worker/src/main/scala/com/lucidchart/piezo/TriggerHistoryModel.scala

```diff
@@ -34,7 +34,15 @@ class TriggerHistoryModel(props: Properties) {
             misfire,
             fire_instance_id
           ) VALUES(?, ?, ?, ?, ?, ?, ?)
-        """.stripMargin
+          ON DUPLICATE KEY UPDATE
+            trigger_name = Values(trigger_name),
+            trigger_group = Values(trigger_group),
+            scheduled_start = Values(scheduled_start),
+            actual_start = Values(actual_start),
+            finish = Values(finish),
+            misfire = Values(misfire),
+            fire_instance_id = Values(fire_instance_id)
+        """.stripMargin,
```

Comment:
Oh this is the problem, my IDE is set up to add the trailing commas with the formatter when I ctrl+s.